CRM/Contribute/BAO/Contribution.php

```diff
@@ -5403,29 +5403,8 @@ protected static function updateMembershipBasedOnCompletionOfContribution($contr
           'membership_activity_status' => 'Completed',
         );
 
-        // CRM-8141 update the membership type with the value recorded in log when membership created/renewed
-        // this picks up membership type changes during renewals
-        // @todo this is almost certainly an obsolete sql call, the pre-change
-        // membership is accessible via $this->_relatedObjects
-        $sql = "
```

Comment:
@omarabuhussein Can you give an example of when this could happen?  I thought from the comments that we can just get the pre-change membership directly.